next/frontend/utils/form.ts

```diff
@@ -19,9 +19,14 @@ import { FormFileUploadFileInfo, FormFileUploadStatusEnum } from '../types/formF
  * of them to ensure consistent behaviour.
  *
  * The default behaviour of RJSF is to prefill all the arrays with minKeys value, with default or
- * `null` value. This make sense for string fields (if there are e.g. 2 required string fields, the
- * library will create 2 empty ones), however we don't want this behaviour as e.g. for files it doesn't
- * make sense (there is nothing such an empty file value).
+ * `null` value. For most of our use-cases this doesn't make sense. As multiple file upload, select
+ * with multiple options or checkbox groups are arrays, e.g. them having minKeys of 1 would make
+ * RJSF to prefill them which would result to `[null]` value which is not correct.
+ *
+ * Unfortunately, there are cases where this behaviour is needed. For example, array fields contain
+ * a list of objects, not prefilling them would require user to manually click "Add" button to add
+ * the first item. This is not a good UX. Therefore, we implemented a patch that allows us to
+ * override this behaviour for specific fields using `overrideArrayMinItemsBehaviour` keyword.
```

Comment:
so the behaviour with overrideArrayMinItemsBehaviour is to populate ? this is not super clear - I understand it overrides the populate:never, but it also sounds like it overrides the default

better naming pehraps ? something like forcePopulateArrayMinItems ?

Reply:
I will add to my personal TODO list.